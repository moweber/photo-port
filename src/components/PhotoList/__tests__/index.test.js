import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import PhotoList from "../";

const categoryName = "commercial";

afterEach(cleanup);

describe("PhotoList is rendering", () => {
  it("renders", () => {
    render(<PhotoList />);
  });

  it("renders", () => {
    const { asFragment } = render(<PhotoList />);
    expect(asFragment()).toMatchSnapshot();
  });
});

it("matches snapshot", () => {
  const { asFragment } = render(<PhotoList category={categoryName} />);
  expect(asFragment()).toMatchSnapshot();
});
