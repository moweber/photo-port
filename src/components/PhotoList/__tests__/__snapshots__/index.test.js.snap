// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoList is rendering renders 1`] = `
<DocumentFragment>
  <div>
    <div
      class="flex-row"
    />
  </div>
</DocumentFragment>
`;

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="flex-row"
    >
      <img
        alt="Grocery aisle"
        class="img-thumbnail mx-1"
      />
      <img
        alt="Grocery booth"
        class="img-thumbnail mx-1"
      />
      <img
        alt="Building exterior"
        class="img-thumbnail mx-1"
      />
      <img
        alt="Restaurant table"
        class="img-thumbnail mx-1"
      />
      <img
        alt="Cafe interior"
        class="img-thumbnail mx-1"
      />
    </div>
  </div>
</DocumentFragment>
`;
